lib/formatter.js

```diff
@@ -319,10 +319,25 @@ function formatOrder(name, traffickerId, partner) {
   };
 }
 
+/**
+ * Generate price points.
+ *
+ * @return {Array}              Price points.
+ */
+var generatePricePoints = function() {
+  var points = [];
+  points.push("0.00");
+  for (var i=1; i<=10000; i++) {
+    points.push((i/100).toFixed(2).toString());
```

Comment:
I don't think you need `.toString()` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed